dashboard/app/models/external.rb

```diff
@@ -23,6 +23,12 @@
 #
 
 class External < DSLDefined
+
+  # Check if the level has a hand-written submit button. Once all submit buttons are removed from markdown, this can go away.
+  def has_submit_button?
+    properties['markdown'].try(:include?, 'next-stage') && properties['markdown'].try(:include?, 'submitButton')
```

Comment:
I don't think submitButton would accidentally get included in level text but next-stage is more likely

"when you are done, click next-stage 5 is the best" or something like that

Is next-stage a class, id, something else? Searching by the specific element attribute is better I think. If you want to be really rigorous, you can translate the md to html and find the element but thats overkill.